alpine-rails-buildkite/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM dkubb/alpine-rails-nginx
+
+# Improve root state, intend to be pushed there
+WORKDIR /
+
+ENV RAILS_ENV=test                  \
+    PGBASE=/var/lib/postgresql      \
+    PGDATA=/var/lib/postgresql/data
+
+COPY etc /etc
+
+RUN rm -r /etc/sv/rails /etc/service/rails \
+ && /root/setup-directories.sh root r /etc/service/postgres /etc/service/buildkite \
+ && /root/setup-directories.sh postgres rw $PGBASE \
+ && ln -s -- /etc/sv/postgres /etc/service/postgres/run \
+ && ln -s -- /etc/sv/buildkite /etc/service/buildkite/run \
+ && echo '@testing-extra https://s3.amazonaws.com/alpine-packages/testing-extra' >> /etc/apk/repositories \
+ && apk add --update-cache imagemagick buildkite-agent@testing-extra phantomjs@testing-extra
+
+USER buildkite
+
+# Setup bundler for buildkite user
+RUN bundle config --global build.nokogiri '--use-system-libraries' \
+ && bundle config --global jobs '8'
+
+USER postgres
+
+# Setup user and database
+RUN pg_ctl initdb --pgdata $PGDATA \
```

Comment:
Actually I dislike that `PGDATA` is set globally, I'd prefer a "local to the dockerfile variable" that does not get globally set. To keep the environment most clean. Right now I traded global infection for `Dockerfile` literal path deduplication. I tend to rever this decision.

If we keep the global setting, we should avoid the redundant `--pgdata` option.